kvirt/cluster/k3s/ctlplanes.yml

```diff
@@ -6,6 +6,9 @@
 {% if extra_ctlplane_disks %}
  {% set extra_disks = extra_ctlplane_disks %}
 {% endif %}
+{% if ctlplane_disk_type %}
```

Comment:
instead of overloading each of the cluster plan files with an extra variable, it's simpler to leverage vmrules and let this as is. That was actually one of the reason why vmrules were created

Reply:
So one would declare disk_type at the vmrules[].root-level? For the nodes for which the disk should not be the GCP pd-standard type disk?

Reply:
yes